back/spec/acceptance/projects_spec.rb

```diff
@@ -916,6 +916,24 @@
           }
         ])
       end
+
+      example 'Downloaded inputs do not include ideas with no content' do
+        native_survey_phase.update!(participation_method: 'ideation')
+
+        # Simulating a survey response with no content, which already
+        # existed before the phase participation_method was changed.
+        survey_response.title_multiloc = {}
+        survey_response.body_multiloc = {}
+        survey_response.save!(validate: false)
```

Comment:
Nitpick: Would probably swap the order of these two updates to simulate more accurately